psi-ms.obo

```diff
@@ -20123,6 +20123,36 @@ def: "Term for a comment field withing the NIST msp file format" [PSI:MS]
 xref: value-type:xsd\:string "The allowed value-type for this CV term."
 is_a: MS:1000499 ! spectrum attribute
 
+[Term]
+id: MS:1003103
+name: ion interpretation format
+def: "Interpretation format used for annotating individual spectrum ion peaks." [PSI:MS]
+is_a: MS:1003078 ! interpreted spectrum attribute
+
+[Term]
+id: MS:1003104
+name: peptide ion interpretation format
+def: "Interpretation format designed primarily for peptides, with allowances for generic chemical formulas and other miscellaneous named ions." [PSI:MS]
+is_a: MS:1003103 ! ion interpretation format
+
+[Term]
+id: MS:1003105
+name: cross-linked peptide ion interpretation format
```

Comment:
How does that compare with XL-MOD? https://github.com/HUPO-PSI/mzIdentML/blob/master/cv/XLMOD.obo